GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
-assembly-versioning-scheme: MajorMinorPatch 
-#next-version: 1.0.0
```

Comment:
I didn't know what used it. It seemed like a no-op to me. I'm using a version.json file via Nerdbank.gitversioning instead, which works locally as well as in the cloud. It seemed to satisfy the requirement more broadly, from my reading of the content of this file.